### ISSUE TITLE
save current learning status

### ISSUE THREAD

**Issue by marziehoghbaie** [keras-team/keras-contrib#546]
Hi, 
I am using the cyclic learning rate scheduler but during training some times I have to pause the training process. Is there any way to load the previous status of the cyclic learning rate instead of starting over?
Thanks
